CHANGELOG.md

```diff
@@ -228,7 +228,7 @@
   [#8712](https://github.com/Kong/kong/pull/8712)
 - **Prometheus**: The plugin doesn't export status codes, latencies, bandwidth and upstream
   healthcheck metrics by default. They can still be turned on manually by setting `status_code_metrics`,
-  `lantency_metrics`, `bandwidth_metrics` and `upstream_health_metrics` respectively.
+  `lantency_metrics`, `bandwidth_metrics` and `upstream_health_metrics` respectively. Because of these flag function will cause the perfmance impact, we recommend using the new `status` plugin to replace it. And now `prometheus` plugin new grafana [dashboard](https://grafana.com/grafana/dashboards/7424-kong-official/) updated
```

Comment:
perfmance -> performance

Reply:
status -> statsd

Reply:
"Enabling those metrics will impact the performance if you have a large volume of Kong entities, we recommend using the statsd plugin with the push model if that is the case."